google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/UpdateBackupRequest.java

```diff
@@ -48,7 +49,22 @@ public UpdateBackupRequest setExpireTime(Instant expireTime) {
     requestBuilder
         .getBackupBuilder()
         .setExpireTime(Timestamps.fromMillis(expireTime.toEpochMilli()));
-    requestBuilder.setUpdateMask(FieldMask.newBuilder().addPaths("expire_time"));
+    requestBuilder.setUpdateMask(fieldMaskBuilder.addPaths("expire_time"));
+    return this;
+  }
+
+  public UpdateBackupRequest setHotToStandardTime(Instant hotToStandardTime) {
+    Preconditions.checkNotNull(hotToStandardTime);
+    requestBuilder
+        .getBackupBuilder()
+        .setHotToStandardTime(Timestamps.fromMillis(hotToStandardTime.toEpochMilli()));
+    requestBuilder.setUpdateMask(fieldMaskBuilder.addPaths("hot_to_standard_time"));
```

Comment:
please avoid magic strings, prefer to use constants & the fieldmask util:

https://github.com/googleapis/java-bigtable/blob/32f244f13d0c8571654d314310a4756fe275d609/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/UpdateInstanceRequest.java#L91